L1Trigger/L1TTrackMatch/plugins/L1TrackJetProducer.cc

```diff
@@ -27,13 +36,12 @@
 #include "Geometry/CommonTopologies/interface/PixelGeomDetUnit.h"
 #include "Geometry/CommonTopologies/interface/PixelGeomDetType.h"
 #include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
-
 #include "DataFormats/L1Trigger/interface/Vertex.h"
 
-#include "L1Trigger/L1TTrackMatch/interface/L1Clustering.h"
```

Comment:
Most of the utility functions that are now in the wrongly named "L1TrackJetEmulatorProducer.h" where poriginally included in the the better named "L1Clustering.h": why aren't you adding the new features in L1Clustering.h?

Reply:
L1Clustering is not descriptive. I moved more functions there. In any case I renamed this file